docs/docker-image-details.md

```diff
@@ -14,28 +14,27 @@ You can deploy the service with the Gravitino Docker image.
 Container startup commands
 
 ```shell
-docker run --rm -d -p 8090:8090 -p 9001:9001 datastrato/gravitino
+docker run --rm -d -p 8090:8090 -p 9001:9001 apache/gravitino
 ```
 
 Changelog
 
-- gravitino:0.5.1
-  - Based on Gravitino 0.5.1, you can know more information from 0.5.1 release notes.
+- apache/gravitino:0.6.0 (Switch to Apache official DockerHub repository)
```

Comment:
You should also change the version to 0.6.0-incubating in this PR, not just here.

Reply:
resolved.